enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -21,174 +21,124 @@ see-also:
 - Replace the SD Splunk forwarder with the CLO which is more flexible and feature-rich.
-- Separate configuration so that service consumer can request management cluster forwarding, but the service provider has final control.
 
 ### User Stories
 
 #### As a service consumer, I want to forward selected API audit logs to my chosen destinations
 
-- configure API audit policy and forwarding using hosted cluster resource.
-- have the desired audit logs forwarded from the management cluster to my destination.
+- Have audit logs forwarded automatically from control-plane CLF to data-plane CLF.
+- Configure API audit policy and forwarding on my data-plane CLF.
+- Forward audit logs in the same way as logs collected from the hosted cluster.
 
 #### As a service provider, I want to forward selected API audit logs to my chosen destinations
+
 - configure the SD internal audit policy
 - configure API audit policy and forwarding using name-spaced logging customer resources in the management cluster.
-  - for example: Red Hat Service Delivery team forwards audit logs to internal splunk destination.
+  - for example: Red Hat Service Delivery team forwards audit logs to internal Splunk destination.
 - independent of user configuration.
 
 ### Goals
 
 - Multiple audit policies can be defined in a management cluster, independent of hosted cluster policies.
 - Multiple audit policies can be defined in a hosted cluster, independent of the management cluster policies.
   See [api-audit-log-policy](/api-audit-log-policy.md) for how the CLO will support this.
-- Allow some forwarding in the _management_ cluster to be configured from the _hosted_ cluster.
-- Keep direct control of management cluster forwarding on the management cluster.
-  - hosted cluster has no direct access to control plane configuration.
-  - data plane resources can request forwarding by the management cluster.
-  - management cluster HLO automatically validates data-plane requests, and enables them if they are safe.
+- Data plane (customer) requests audit log forwarding by modifying data-plane CLF configuration, forwarding from control plane starts automatically.
 
 ### Non-Goals
 
 - This proposal only covers API audit logs. No change to application, infrastructure or node audit logs.
-- Management cluster is not _required_ to honour all possible forwarding configurations allowed by CLF. \
 
 ## Proposal
 
 ### Workflow Description
 
 #### Hosted Cluster, data plane
 
-1. Create a `Policy.audit.k8s.io` resource with the desired policy
-   (see [api-audit-log-policy](/api-audit-log-policy.md))
-1. Create a `HypershiftLogForwarder` (HLF) resource with an `audit` input linked to the policy.
-1. Create HLF `outputs` and `pipelines` to forward the resulting logs.
-1. Optionally create a `ClusterLogForwarder` to forward other logs (not API-audit) as usual.
+1. Install the Cluster Logging Operator (CLO).
+1. Create a `ClusterLogForwarder` with a HTTP intupt.
```

Comment:
Are these installed / setup for every hostedcluster , or opt in by a customer at install time or day-2 ?

Reply:
The data-plane CLF is opt-in by the customer on day 2. The control-plane CLF is created automatically by the HLO in response to the customer CLF existing and having the relevant configuration.